receiver/hostmetricsreceiver/testdata/config.yaml

```diff
@@ -0,0 +1,17 @@
+receivers:
+  hostmetrics:
```

Comment:
Please add one more hostmetrics receiver with a name, for e.g. 'hostmetrics/customname'. Similar to [otlp](https://github.com/open-telemetry/opentelemetry-collector/blob/2759f27be08887ccd0a31e69d30d7d8bc0840756/receiver/otlpreceiver/testdata/config.yaml#L4-L9)
typical use would be that one scraper is configured with 10 second interval collecting memory metrics and another one configured with 60 second interval collecting cpu metrics. Unless each category allows configuring separate interval. In that case two instance of hostmetrics should be disallowed.